src/cat.ts

```diff
@@ -100,6 +115,38 @@ export class Cat {
     return this._zetas;
   }
 
+  public get prior() {
+    return this._prior;
+  }
+
+  private static validatePrior(priorDist: string, priorPar: number[], minTheta: number, maxTheta: number) {
+    if (priorDist !== 'norm' && priorDist !== 'unif') {
+      throw new Error('The prior distribution you provided is not supported');
+    }
+    if (priorDist === 'norm') {
+      if (priorPar.length !== 2) {
+        throw new Error('The prior distribution parameters you provided are not valid');
+      }
+      if (priorPar[1] <= 0) {
+        throw new Error('The prior distribution standard deviation you provided is not valid');
```

Comment:
If you can, it's always good practice to tell users what your expected and received in an error message.
```suggestion
        throw new Error(`Expected a positive prior distribution standard deviation. Received ${priorPar[1]}`);
```